src/extractor/guidance/turn_discovery.cpp

```diff
@@ -66,12 +68,25 @@ bool findPreviousIntersection(const NodeID node_v,
     // if the edge is not traversable, we obviously don't have a previous intersection or couldn't
     // find it.
     if (node_based_graph.GetEdgeData(result_via_edge).reversed)
+    {
+        result_via_edge = SPECIAL_EDGEID;
+        result_node = SPECIAL_NODEID;
```

Comment:
since it only makes some turn lane cases (very rare) break, it is not a urgent fix. Still one of the reasons I want to have it in 5.4